<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>496</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Семейное древо</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QListWidget" name="recordsListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>9</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelFio">
            <property name="text">
             <string>ФИО:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="FioEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Родители</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="labelFather">
               <property name="text">
                <string>Отец:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBoxFatherFio">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>не задан</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelMather">
               <property name="text">
                <string>Мать:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBoxMatherFio">
               <item>
                <property name="text">
                 <string>не задана</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelBirthday">
            <property name="text">
             <string>Дата рождения:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateEdit" name="dateEditBirthday">
            <property name="minimumDate">
             <date>
              <year>1752</year>
              <month>9</month>
              <day>14</day>
             </date>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkIsAlive">
            <property name="text">
             <string>Жив</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelDeathday">
            <property name="text">
             <string>Дата смерти:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateEdit" name="dateEditDeathday"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelGender">
            <property name="text">
             <string>Пол:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBoxGender">
            <item>
             <property name="text">
              <string>Муж.</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Жен.</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelCitizenship">
            <property name="text">
             <string>Гражданство:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="comboBoxCitizenship">
            <item>
             <property name="text">
              <string>Россия</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Украина</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Польша</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Германия</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>США</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Израиль</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Япония</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Кипр</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QCheckBox" name="checkSlughil">
            <property name="text">
             <string>Состоял(а) на военной службе</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButtonCreate">
            <property name="text">
             <string>Создать запись</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSave">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fill">
            <property name="text">
             <string>Заполнить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="CreateDB"/>
    <addaction name="OpenDB"/>
    <addaction name="SaveDB"/>
    <addaction name="SaveAsDB"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="CreateDB">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="OpenDB">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="SaveDB">
   <property name="text">
    <string>Сохрнить</string>
   </property>
  </action>
  <action name="SaveAsDB">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
